modules/loader/src/main/java/com/opengamma/strata/loader/csv/CsvLoaderColumns.java

```diff
@@ -46,24 +46,28 @@
   /** CSV header. */
   public static final String DIRECTION_FIELD = "Direction";
   /** CSV header. */
-  static final String LEG_1_DIRECTION_FIELD = "Leg 1 " + DIRECTION_FIELD;
+  public static final String LEG_1_DIRECTION_FIELD = "Leg 1 " + DIRECTION_FIELD;
```

Comment:
I forgot to set these as public when I added the field in my previous merge request, a couple weeks ago.